zjit/src/options.rs

```diff
@@ -155,6 +163,21 @@ fn update_profile_threshold(options: &Options) {
     }
 }
 
+/// Print YJIT options for `ruby --help`. `width` is width of option parts, and
+/// `columns` is indent width of descriptions.
+#[unsafe(no_mangle)]
+pub extern "C" fn rb_zjit_show_usage(help: c_int, highlight: c_int, width: c_uint, columns: c_int) {
+    for &(name, description) in ZJIT_OPTIONS.iter() {
+        unsafe extern "C" {
+            fn ruby_show_usage_line(name: *const c_char, secondary: *const c_char, description: *const c_char,
+                                    help: c_int, highlight: c_int, width: c_uint, columns: c_int);
+        }
+        let name = CString::new(name).unwrap();
+        let description = CString::new(description).unwrap();
```

Comment:
assert 80 cols here?

Reply:
The limitation is more tightly coupled to `show_usage_part` that formats each line with indentation (we can't even pass 80 cols of `description` from this function, for example). I would not add such an assertion on this side.